cms/src/components/content/EditContent.spec.ts

```diff
@@ -64,6 +64,38 @@ describe("EditContent.vue", () => {
         });
     });
 
+    it("reverts changes to the original content", async () => {
+        const wrapper = mount(EditContent, {
+            props: {
+                docType: DocType.Post,
+                id: mockData.mockPostDto._id,
+                languageCode: "eng",
+            },
+        });
+
+        await waitForExpect(() => {
+            const titleInput = wrapper.find('input[name="title"]');
+            expect((titleInput.element as HTMLInputElement).value).toBe(
+                mockData.mockEnglishContentDto.title,
+            );
+        });
+
+        const titleInput = wrapper.find('input[name="title"]');
+        await titleInput.setValue("New Title");
```

Comment:
Can you try this: `await wrapper.find('input[name="title"]`).setValue("New Title");